service_info_cms/cms_menus.py

```diff
@@ -25,4 +25,23 @@ def modify(self, request, nodes, namespace, root_id, post_cut, breadcrumb):
             self._modify_nodes(nodes)
         return nodes
 
+
+class AddRatingExtension(Modifier):
+    """Make the RatingExtension available in the menu."""
```

Comment:
No, that menu item works fine with this code disabled.  This "menu" code deals with rendering the navigation menus that users see.